eng/pipelines/docs.yml

```diff
@@ -0,0 +1,50 @@
+trigger:
+  - master
+
+variables:
+  NodeVersion: "10.x"
+
+jobs:
+  - job: "DocGen"
+    variables:
+      skipComponentGovernanceDetection: true
+    timeoutInMinutes: 120
+    pool:
+      vmImage: "vs2017-win2016"
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "$(NodeVersion)"
+        displayName: "Install Node.js $(NodeVersion)"
+
+      - script: |
+          npm install -g npm@6.9.0
+        displayName: "Install npm version 6.9.0"
+
+      - script: |
+          node common/scripts/install-run-rush.js install
+        displayName: "Install dependencies"
+
+      - script: |
+          npm i -g typedoc
+        displayName: "Install typedoc"
+
+      - script: |
+          npm install nunjucks
+        displayName: "Install nunjucks"
```

Comment:
Nit: don't need two steps correct?

Reply:
I think i can combine it to one step and call it "Install nunjucks and fs-extra"
I will still test it once to be sure.

Reply:
just tested this on playground, seems like i'll need separate steps to install each package, otherwise it fails to install the 2nd one @scbedd

Reply:
Hmm. Ok.